test/e2e/testing-manifests/storage-csi/hostpath/hostpath-v0/csi-hostpath-snapshotter.yaml

```diff
@@ -0,0 +1,48 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-snapshotter
+  labels:
+    app: csi-snapshotter
+spec:
+  selector:
+    app: csi-snapshotter
+  ports:
+  - name: dummy
+    port: 12345
+
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-snapshotter
+spec:
+  serviceName: "csi-snapshotter"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-snapshotter
+  template:
+    metadata:
+      labels:
+        app: csi-snapshotter
+    spec:
+      serviceAccount: csi-snapshotter
+      containers:
+      - name: csi-snapshotter
+        image: quay.io/k8scsi/csi-snapshotter:v1.0.1

```

Comment:
This file is under test/e2e/testing-manifests/storage-csi/hostpath/hostpath-v0, so we should use quay.io/k8scsi/csi-snapshotter:v0.4.1.  Otherwise it is a mismatch with other yaml files.

Reply:
updated